build/docker/Release.dockerfile

```diff
@@ -12,15 +12,22 @@ FROM alpine:latest
 ARG VERSION
 ARG LOCATION
 
-LABEL maintainer="Vault Team <vault@hashicorp.com>"
-LABEL version=$VERSION
+LABEL name="Vault K8s" \
```

Comment:
Maybe worth a quick comment to mention which of these are used by RedHat certification, for future editors.

Reply:
This file isn't subject to RedHat certification, only the UBI image, so it was omitted. I added the labels just to make them similar.

Reply:
Oh good point, same comment on the other Dockerfile then :)

Reply:
Added a generic blurb anyway 👍